core/src/main/java/hudson/Proc.java

```diff
@@ -510,7 +510,7 @@ public OutputStream getStdin() {
     /**
     * An instance of {@link Proc}, which has an internal workaround for JENKINS-23271.
     * It presumes that the instance of the object is guaranteed to be used after the {@link Proc#join()} call.
-    * See <a href="https://jenkins-ci.org/issue/23271">JENKINS-23271</a>
```

Comment:
FWIW `jenkins.io/issue/` is the redirect URL for links to JENKINS Jira issues. Unless it's been deprecated, maybe it would make sense to extend its use? (Or perhaps it's time to abandon it entirely.)